README.md

```diff
@@ -109,6 +110,13 @@ Perform a simple single warmup request (per worker) before starting
 execution. Note that this will be reflected in the counters that
 Nighthawk writes to the output. Default is false.
 
+--request-source-plugin-config <string>
+Request Source plugin configuration in json or compact yaml. Mutually
```

Comment:
Request Source is not meaningful to the average user of nighthawk.  Should we point them to request source documentation?

In other words - If someone doesn't know what request source is, how would they know to use this feature?